app/routes/course/assignment.ts

```diff
@@ -0,0 +1,73 @@
+import { INTERNAL_SERVER_ERROR, OK } from 'http-status-codes';
+import * as Router from 'koa-router';
+import { AssignmentModel, IApiResponse, IAssignmentModel, TaskModel } from '../../models';
+import { setResponse } from '../utils';
+
+export const courseAssignmentGetRoute = async (ctx: Router.IRouterContext) => {
+    try {
+        const { courseId, studentId } = ctx.params;
+        const assignments = await AssignmentModel.find({
+            courseId,
+            studentId,
```

Comment:
We mustn't send userId in request! We use koa-passport and we can get userId just using this object ctx.state.user.